public/app/features/logs/components/LogRowMessage.tsx

```diff
@@ -77,6 +77,15 @@ const getStyles = (theme: GrafanaTheme2, showContextButton: boolean, isInDashboa
       right: ${isInDashboard ? '40px' : `calc(75px + ${theme.spacing()} + ${showContextButton ? '80px' : '40px'})`};
       margin-top: -${theme.spacing(0.125)};
     `,
+    logLine: css`
+      background-color: transparent;
+      border: none;
+      diplay: inline;
+      font-family: ${theme.typography.fontFamilyMonospace};
+      font-size: ${theme.typography.bodySmall.fontSize};
+      letter-spacing: ${theme.typography.bodySmall.letterSpacing};
```

Comment:
As soon as log lines are wrapping, they seem to be centered:
![image](https://user-images.githubusercontent.com/8092184/208892323-fa350c4d-5091-4bcb-b150-448855d001ec.png)

Could be fixed with:

```suggestion
      letter-spacing: ${theme.typography.bodySmall.letterSpacing};
      text-align: left;
```

Reply:
Oh, good catch!